commands/memberCommand.go

```diff
@@ -0,0 +1,49 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/Strum355/log"
+
+	"github.com/UCCNetsoc/discord-bot/config"
+	"github.com/bwmarrin/discordgo"
+	"github.com/spf13/viper"
+)
+
+func members(ctx context.Context, s *discordgo.Session, m *discordgo.MessageCreate) {
+	args := strings.Fields(strings.TrimPrefix(m.Content, viper.GetString("bot.prefix")+"members"))
+
+	servers := viper.Get("discord.servers").(*config.Servers)
+	// publicServer, err := s.Guild(servers.PublicServer)
+
+	var num int
```

Comment:
Would be more readable to move this initialization down to above the for loop where it's used